.github/workflows/swift.yml

```diff
@@ -47,7 +47,8 @@ jobs:
 
   unittests: 
     if: "${{ github.event.inputs.PREP == '' && github.event.inputs.RELEASE == '' }}" 
-    uses: optimizely/swift-sdk/.github/workflows/unit_tests.yml@master
+    ##uses: optimizely/swift-sdk/.github/workflows/unit_tests.yml@master
+    uses: optimizely/swift-sdk/.github/workflows/unit_tests.yml@jae/empty-odp-action
```

Comment:
Looks good overall, can we update to master before merging

Reply:
I'll update it with the next PR coming :)